docs/community/dev-card.md

```diff
@@ -0,0 +1,17 @@
+---
+id: dev-card
+title: 'Showcase your open source stats with your dev card'
+sidebar_label: 'Dev Card'
+keywords:
+- dev-card 
+---
+
+The Dev card feature is what we like to call your "open source business card." At the front, you see your username, the number of pull requests you have created, repositories you contribute to, a `View Profile` button that takes you to your dashboard, and a green icon, which describes your activity rate.
+[screenshot of front of a dev card](../../static/img/Group%20113.svg)
```

Comment:
Instead of linking the images, display them on the page.

Reply:
How do I go about doing that @BekahHW?
Like this?:
```
<img src="">
```

Reply:
You can do that or add an ! in front of the markdown format ![alt description](image path)

Reply:
Ahh ok

Reply:
Done @BekahHW